src/main/java/com/dev/cinema/dao/interfaces/CinemaHallDao.java

```diff
@@ -7,4 +7,6 @@
     public CinemaHall add(CinemaHall cinemaHall);
 
     public List<CinemaHall> getAll();
+
+    CinemaHall getCinemaHallById(Long id);
```

Comment:
```suggestion
    CinemaHall getById(Long id);
```
з назви інтерфейсу зрозуміло що саме повинен повертати даний метод